config/default.yml

```diff
@@ -38,8 +38,32 @@ Style/EmptyElse:
 Style/Documentation:
   Enabled: no
 
+Naming/MemoizedInstanceVariableName:
+  EnforcedStyleForLeadingUnderscores: optional
+
+Metrics/MethodLength:
+  Max: 15
+
+Metrics/BlockLength:
+  Enabled: false
+
 RSpec/NoExpectationExample:
   Enabled: no
 
 RSpec/ExampleLength:
   Enabled: false
+
+RSpec/MultipleMemoizedHelpers:
+  Enabled: false
+
+RSpec/MultipleExpectations:
+  Enabled: false
+
+RSpec/NestedGroups:
+  Max: 4
+
+RSpec/MessageSpies:
+  Enabled: false
+
+RSpec/IndexedLet:
+  Max: 2
```

Comment:
When wanting to test order, then 2 is sufficient and there really aren't any better names than record1, record2 or first_record, second_record